### DISCUSSION TITLE
Display condition aliases in trace step details

### DISCUSSION THREAD

**Discussion by kadaan** [home-assistant/frontend#17993] [Localization]
Reading the `Step Details` in the automation trace is difficult for complex automations because the conditions are referred to via condition path like `conditions/0/conditions/1/conditions/0/entity_id/0`.  If the condition alias was displayed in addition the the condition path it would be much easier to diagnose which conditions are not working.

**Comment by ildar170975** [comment]
Same should be for other script sections which may have aliases.
